### PR TITLE
updated starling transactions to v2

### PR DESCRIPTION
**Pull request by whiteshadow-gr**

### CODE REVIEW

File: dataplug-starling/app/org/hatdex/dataplugStarling/apiInterfaces/StarlingAccountInterface.scala [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    val mailer: Mailer,
+    val scheduler: Scheduler,
+    val provider: StarlingProvider) extends DataPlugEndpointInterface with RequestAuthenticatorOAuth2 {
+
+  val namespace: String = "starling"
+  val endpoint: String = "accounts"
+  protected val logger: Logger = Logger(this.getClass)
+
+  val defaultApiEndpoint: ApiEndpointCall = StarlingAccountInterface.defaultApiEndpoint
+  val refreshInterval: FiniteDuration = 1.hour
```
Comment (augustinas) [inline dataplug-starling/app/org/hatdex/dataplugStarling/apiInterfaces/StarlingAccountInterface.scala]: 7 days sounds more realistic

**Comment by whiteshadow-gr** [comment]
Maybe update the starling model in the DB? To show the correct logo as well since it's in the project. It's probably wrong URL in the model

**Comment by augustinas** [comment]
> Maybe update the starling model in the DB? To show the correct logo as well since it's in the project. It's probably wrong URL in the model

Of course :D

**Comment by whiteshadow-gr** [comment]
I didn't make the changes??? Just noticed this PR

**Comment by augustinas** [comment]
Work on the Starling data plug was discontinued.

**Pull request closed without merging**